server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/functions.js

```diff
@@ -339,7 +339,7 @@ function doLoggedPostCall(call, callback, shouldSanitize) {
  * stores it on a sessionStorage variable
  */
 function getManager() {
-  return getJSONForTable(contextPath + 'rest/manager', 'manager');
+  return getJSONForTable(REST_V2_PREFIX + '/manager', 'manager');
```

Comment:
`REST_V2_PREFIX` is defined as `const REST_V2_PREFIX = contextPath + 'rest-v2';` on line 465. 
I didn't see any console errors but that constant definition should probably get moved to the top of this file for readability.